libs/httpsignature/nitro.go

```diff
@@ -0,0 +1,80 @@
+package httpsignature
+
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/brave-intl/bat-go/libs/nitro"
+	"github.com/hf/nitrite"
+)
+
+// NitroSigner is a placeholder struct to sign using a nitro attestation
+type NitroSigner struct{}
```

Comment:
I'm not seeing where we're using any of this yet. Is the intention to just integrate it this way now and then we'll have a follow up PR to integrate it into the nitro-settlements-dev branch? That would be ideal if so, but just want to verify my assumption.